relay-server/src/actors/store.rs

```diff
@@ -1038,6 +996,20 @@ pub struct StoreEnvelope {
     pub scoping: Scoping,
 }
 
+// Envelope for the possible Messages that can be sent to the StoreForwarder
```

Comment:
May as well make this a doc comment (and end it in a full stop).  One way to think of this is whether running cargo doc with `--document-private-items` will give you usable output.  Here that would indeed improve the doc.